test/Microsoft.AspNetCore.Mvc.FunctionalTests/DefaultOrderTest.cs

```diff
@@ -26,19 +24,19 @@ public DefaultOrderTest(MvcTestFixture<BasicWebSite.Startup> fixture)
         public HttpClient Client { get; }
 
         [Theory]
-        [InlineData(typeof(IActionDescriptorProvider), typeof(ControllerActionDescriptorProvider), -1000)]
+        [InlineData(typeof(IActionDescriptorProvider), "Microsoft.AspNetCore.Mvc.ApplicationModels.ControllerActionDescriptorProvider", -1000)]
```

Comment:
@JamesNK this case is a bit different, unit test projects see all the `internal`s but functional tests do not.

Reply:
This specific test is kinda awful - it's a functional test, for what can be written as a unit test. Additionally, I'm not entirely sure what the benefit of this test is - it verifies if the `Order` property on these types are set - we rarely test those since they nearly never change: https://github.com/aspnet/Mvc/blob/release/2.2/test/WebSites/BasicWebSite/Controllers/OrderController.cs#L13-L36

Reply:
I'm planning on removing these tests in a follow up.